docs/samples/Microsoft.ML.Samples/Dynamic/OneHotEncodingTransform.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.OneHotEncodingTransformer;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class OneHotEncodingTransform
+    {
+        public static void Example()
```

Comment:
>Example [](start = 27, length = 7)

please link it to the extension it documents through the <example> node. See the other extension methods xml doc.